spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -476,16 +440,15 @@ public class TransformProcessor {
 }
 ----
 
-NOTE: It is important to understand that, when you consume from the same binding by using `@StreamListener`, a pub-sub model is used. Each method annotated with `@StreamListener` receives its own copy of the message and each one has its own consumer group.
-However, if you share a bindable channel as an input for `@Aggregator`, `@Transformer`, or `@ServiceActivator`, those consume in a competing model. No individual consumer group is created for each subscription.
+NOTE: While this may be skipping ahead a bit, note that it is important to understand that, when you consume from the same binding using `@StreamListener` annotation, a pub-sub model is used.
+Each method annotated with `@StreamListener` receives its own copy of a message and each one has its own consumer group.
+However, if you consume from the same binding using one of the Spring Integration annotation such as `@Aggregator`, `@Transformer`, or `@ServiceActivator`, those consume in a competing model.
```

Comment:
Replace with "However, if you consume from the same binding by using one of the Spring Integration annotation (such as `@Aggregator`, `@Transformer`, or `@ServiceActivator`), those consume in a competing model."